NewFEUBot.py

```diff
@@ -0,0 +1,79 @@
+import discord
+import os
+from discord.ext import commands
+import sys, traceback
+from sys import argv
+from dotenv import load_dotenv
+import re
+import random
+import urllib
+import urllib.request
+import urllib.error
+import json
+from feubotFormatter import FeubotFormatter
+
+bot = commands.Bot(command_prefix =  "!")
+
+def trunc_to(ln, s):
+    if len(s) >= ln: return s
+    else: return s[:ln-3] + "..."
+
+
+def feuSearchEmbed(posts, threads, term):

```

Comment:
the convention used by the rest of the codebase is `lowercase_with_underscores`, so it's a bit jarring to have a `camelCase` function here

Reply:
fixed